packages/web-components/fast-components/src/skeleton/fixtures/base.html

```diff
@@ -77,18 +93,14 @@ <h4>Used as element blocks with shimmer effect element</h4>
 
     <h4>Using SVG via Pattern attribute</h4>
     <fast-skeleton
-        style="border-radius: 4px; width: 500px; height: 250px;"
+        class="card-pattern"
         shape="rect"
         pattern="https://static.fast.design/assets/skeleton-test-pattern.svg"
```

Comment:
This file is located at `sites/site-utilities/statics/assets/`, you can remove the hard coded fill there.

Reply:
I believe this was hardcoded to be #ffffff by design. @litteredwitherrors is this correct?